src/components/GameboardOnline/GameboardOnline.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+import { getAuth } from "firebase/auth";
+import { gameIsOver, updateGameOnline } from "../../services/game.service";
+import { PlayCard } from "../PlayCard/PlayCard";
+
+const turnIsAllowed = (turn) => turn < 2;
+
+const endOfGame = (board) => {
+  return board.every((card) => card.cleared);
+};
+
+const getNextPlayer = ({ players, currentPlayer }) => {
+  const playerArray = Object.keys(players);
```

Comment:
könnte nur players heißen

Reply:
players ist in dem kontext leider schon definiert, aber stimme zu, der name ist verbesserungswürdig